internal/cluster/clustersetup/activity_install_ingress_controller.go

```diff
@@ -32,13 +33,31 @@ import (
 	"github.com/banzaicloud/pipeline/src/dns"
 )
 
-type ingressControllerValues struct {
-	Traefik traefikValues `json:"traefik"`
+const InstallIngressControllerActivityName = "install-ingress-controller"
+
+type InstallIngressControllerActivity struct {
+	config      clusterconfig.LabelConfig
+	helmService HelmService
 }
 
-type traefikValues struct {
-	SSL     sslTraefikValues     `json:"ssl"`
-	Service serviceTraefikValues `json:"service,omitempty"`
+// NewInstallIngressControllerActivity returns a new InstallIngressControllerActivity.
+func NewInstallIngressControllerActivity(
+	config clusterconfig.LabelConfig,
+	helmService HelmService,
+) InstallIngressControllerActivity {
+	return InstallIngressControllerActivity{
+		config:      config,
+		helmService: helmService,
+	}
+}
+
+type InstallIngressControllerActivityInput struct {
+	ClusterID, OrgID uint
```

Comment:
**Optional**: could be opinionated, but IMO multiple field declarations on a single line decrease readability with no benefit (we spared one extra line in a file with 162 other lines).